### ISSUE TITLE
Huntress's Bow does not upgrade with level-ups.

### ISSUE THREAD

**Issue by Moontone** [00-Evan/shattered-pixel-dungeon#595]
* **Expected**: bow gains +1 to it's level each time Huntress gains new level.
* **In fact**: Huntress is already 4-th level, but the bow is at it's basic level, just indicating that huntress got +1 stronger (drank potion of strength)
* **How to reproduce**: Just play the game by Huntress and get some level-ups

**Comment by TheYsconator** [comment]
Looking at the code of the `SpiritBow`, you can see that the bow does not gain a level on each Huntress' level:

```java
@Override
public int level() {
    return (Dungeon.hero == null ? 0 : Dungeon.hero.lvl/5) + (curseInfusionBonus ? 1 : 0);
}
```

**Comment by Moontone** [comment]
> Looking at the code of the `SpiritBow`, you can see that the bow does not gain a level on each Huntress' level:
> 
> ```java
> @Override
> public int level() {
>     return (Dungeon.hero == null ? 0 : Dungeon.hero.lvl/5) + (curseInfusionBonus ? 1 : 0);
> }
> ```
> 
> In other words: when talking about expected functionality, the bow is behaving as intended so far. Does it state somewhere explicitly that the bow gains a level on each level the Huntress gains?

"The bow steadily grows stronger as the huntress levels up" - from the game.

**Comment by TheYsconator** [comment]
Hmm yeah, technically not incorrect, but I can see it may be confusing/ underwhelming. However, I tried coming up with better alternatives for you, but I was not able to come up with any that were both significantly better and that were in line with (what I believe is) the style of the game. My whole reasoning was a couple of paragraphs, which I spared you in the end - mostly because it came down to 'my vision of Evan's game', which is by far not as interesting as 'Evan's vision of his own game'.

In any case; I agree it's a tad quirky, but not nearly enough to warrant spending more of my time on. (The best alternative I've been able to come up with would be "The bow's strength seems linked to that of its wielder", but it is in my opinion not a significant improvement, if an improvement at all.)
